stories/core_components/ComboChart.tsx

```diff
@@ -297,6 +304,29 @@ storiesOf("Core components/ComboChart", module)
             </ScreenshotReadyWrapper>,
         ),
     )
+    .add("stack primary measures has many data with 1 VIEW BY and chart type is COLUMN", () =>
+        screenshotWrap(
```

Comment:
Just two small questions regarding the screenshot image for this storybook (https://github.com/gooddata/gooddata-react-components/pull/1067/files?short_path=9ac6f84#diff-9ac6f84d777981144e1f4ace43abb887) 
1. Is this image the case when without the fix there would be many overlapping labels? 
2. Is it correct that there are no labels now?

Reply:
1 & 2: Yes I add story book image base on use case in bug, for column-column if label is overlap We will hidden

Reply:
Thanks :+1: